templates/admin/tool_shed_repository/initiate_repository_installation.mako

```diff
@@ -17,11 +17,11 @@
             // Make ajax call
             $.ajax( {
                 type: "POST",
-                url: "${h.url_for( controller='admin_toolshed', action='manage_repositories' )}",
+                url: "/api/tool_shed_repositories/initiate_repository_installation",
```

Comment:
Will this work if Galaxy is served on a prefix - I feel like this was using url_for a reason.